### PR TITLE
Return previously failing custom action test

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Actual fix was this: https://github.com/opencrvs/infrastructure/commit/5564dd2cbd137c19acd46a63214003a880d39b03

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/api/notification/informantNotification.ts [hunk 1]
```diff
@@ -183,7 +183,7 @@ async function getNotificationParams(
     }
   }
 
-  throw new Error(`Invalid action type "${pendingAction.type}"`)
+  return
```
Comment (cibelius) [inline src/api/notification/informantNotification.ts:186]: It felt unnecessary to throw an error, instead lets just return undefined and not send a notification

**Review by Zangetsu101 — APPROVED** [review]
